l10n_it_pec_messages/wizard/mail_compose_message.py

```diff
@@ -68,26 +68,15 @@ def get_message_data(self, cr, uid, message_id, context=None):
         if context is None:
             context = {}
         if context.get('reply_pec'):
-            message_data = self.pool.get('mail.message').browse(cr, uid, message_id, context=context)
-
-            # create subject
-            re_prefix = _('Re:')
-            reply_subject = tools.ustr(message_data.subject or message_data.record_name or '')
-            if not (reply_subject.startswith('Re:') or reply_subject.startswith(re_prefix)) and message_data.subject:
-                reply_subject = "%s %s" % (re_prefix, reply_subject)
```

Comment:
@eLBati i, ricordo che avevo fatto delle prove e c'era qualche anomalia nel fare il super.. infatti lo avevo mezzo come condizione else, ma se lo hai provato e funziona sia in invio normale che in invio risposte, per me ok..